### ISSUE TITLE
Wrong page title

### ISSUE THREAD

**Issue by Farfurix** [DevExpress/testcafe-hammerhead#2374]
Found in https://github.com/DevExpress/testcafe/issues/5272.

Page: https://phenixrts.com/channel/#phenixWebsiteDemo

Native/testcafe@1.8.7 differences:
![Capture](https://user-images.githubusercontent.com/30019338/86234571-d078e700-bb9f-11ea-8408-7056c96256ad.PNG)

testcafe-hammerhead@17.1.8:
![Capture](https://user-images.githubusercontent.com/30019338/86235098-9825d880-bba0-11ea-97ea-44783520de88.PNG)

**Comment by miherlosev** [comment]
![origin_web_site](https://user-images.githubusercontent.com/4133518/86609288-7075ac80-bfb4-11ea-89a3-8ab2873da60c.png)

**Comment by miherlosev** [comment]
Simplified example:
```js
<html>
    <head>
        <meta charset="utf-8">
        <title><%= title %></title>
        <script>
            var titleElement = document.getElementsByTagName('title');

            // use injected value or replace with default
            if (titleElement.length > 0 && (titleElement[0].innerHTML === '&lt;%= title %&gt;' || !titleElement[0].innerText)) {
                titleElement[0].innerText = 'Channel';
            }
        </script>
    </head>
    <body>
        <div id="app"></div>
	</body>
</html>	
```

**Comment by AndreyBelym** [comment]
Closed by #2378.
